akka-cluster/src/multi-jvm/scala/akka/cluster/ClusterDeathWatchSpec.scala

```diff
@@ -73,6 +73,9 @@ abstract class ClusterDeathWatchSpec
     expectMsgType[ActorIdentity].ref.get
   }
 
+  // only assigned on node 1
```

Comment:
```suggestion
  // only assigned on node 5
```

Reply:
No, that is correct, assignment in `runOn(first)` below